src/main/flight/alt_hold.c

```diff
@@ -101,20 +98,21 @@ void altHoldUpdateTargetAltitude(void)
     // - the pilot has control nice control over altitude, and the deadband is wide
     // - Slow controlled descents are possible, eg for landing
     // - fine-tuning height is possible, eg if there is slow sensor drift
-    // - to keep the craft stable, throttle must be in the deadband, making exits smoother
-
-    const float rcThrottle = rcCommand[THROTTLE];
-
-    const float lowThreshold = 0.5f * (autopilotConfig()->hover_throttle + PWM_RANGE_MIN); // halfway between hover and MIN, e.g. 1150 if hover is 1300
-    const float highThreshold = 0.5f * (autopilotConfig()->hover_throttle + PWM_RANGE_MAX); // halfway between hover and MAX, e.g. 1650 if hover is 1300
 
     float throttleAdjustmentFactor = 0.0f;
-    if (rcThrottle < lowThreshold) {
-        throttleAdjustmentFactor = scaleRangef(rcThrottle, PWM_RANGE_MIN, lowThreshold, -1.0f, 0.0f);
-    } else if (rcThrottle > highThreshold) {
-        throttleAdjustmentFactor = scaleRangef(rcThrottle, highThreshold, PWM_RANGE_MAX, 0.0f, 1.0f);
+    if (altHoldState.allowStickAdjustment && calculateThrottleStatus() != THROTTLE_LOW) {
```

Comment:
added the conditional check for THROTTLE_LOW here so that it's easy to quickly get stable altitude control in an emergency